ghost/core/core/frontend/helpers/get.js

```diff
@@ -357,7 +357,9 @@ module.exports = async function get(resource, options) {
                         apiOptions,
                         returnedRows: returnedRowsCount
                     }
-                }));
+                }), {
+                    time: totalMs
+                });
```

Comment:
I've added this as a separate arg instead of adding it to the error as the error type does not allow it (and it does not make sense to add this field to the error type). The arg should get merged into the overall log when being processed (see [here](https://github.com/TryGhost/framework/blob/main/packages/logging/lib/GhostLogger.js#L502-L512))